src/BlockTypes/MiniCart.php

```diff
@@ -371,7 +371,8 @@ protected function get_markup( $attributes ) {
 		</span>';
 
 		if ( is_cart() || is_checkout() ) {
-			return '<div class="' . $wrapper_classes . '">
+			// It is not necessary to load the Mini Cart Block on Cart and Checkout page.
+				return '<div class="' . $wrapper_classes . '" style="visibility:hidden" aria-hidden="true">
 				<button class="wc-block-mini-cart__button ' . $classes . '" aria-label="' . esc_attr( $aria_label ) . '" style="' . $style . '" disabled>' . $button_html . '</button>
 			</div>';
 		}
```

Comment:
I'm honestly more on the camp of not rendering anything because the mini cart would be occupying limited real estate and the layout would look a bit strange.

I think this highly depends on the layout of the header and I wonder if it can be solved via CSS only.

I don't see it as a priority bug because it can be fixed with CSS.
![IMG_0449](https://user-images.githubusercontent.com/6165348/186547748-622eaa15-5f91-42b9-89fd-67b2e4b3f6cc.jpg)
![IMG_0450](https://user-images.githubusercontent.com/6165348/186547753-f3a0b563-b779-4e10-8875-78540a66eb26.jpg)

Reply:
`visibility:hidden` was added for [this reason](https://github.com/woocommerce/woocommerce-blocks/pull/5996#pullrequestreview-900342669). As you said, it depends on the layout. I'm not sure that we will be able to find a solution that has a good fit for all layouts.

Reply:
Yes, I was just raising what I consider an equally valid issue and that you can't have it all :/ somehow a layout will be broken and we should optimize for the most popular one (default) which is the one Albert showed.